### PR TITLE
fix #772 app language activity landscape

### LINKED ISSUE #772: Lowfi Landscape: AppLanguageActivity
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/2b8387b0-5ec6-4e42-bffa-c039128b2f12/OP-App-Language-/

`AppLanguageActivity` should support landscape mode. Also add test cases for landscape mode too.

This issue is currently blocked because the landscape mode mocks are not ready yet.

### PR DESCRIPTION
**Pull request by Kushal-Gera**
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +150,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (aggarwalpulkit596) [inline .idea/codeStyles/Project.xml:153]: Don't push changes made into the system files

File: .idea/gradle.xml [hunk 2]
```diff
@@ -19,4 +21,4 @@
       </GradleProjectSettings>
     </option>
   </component>
-</project>
+</project>
```
Comment (aggarwalpulkit596) [inline .idea/gradle.xml:24]: Ditto

File: .idea/misc.xml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="JDK" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (aggarwalpulkit596) [inline .idea/misc.xml:9]: Ditto

File: build.gradle [hunk 4]
```diff
@@ -9,7 +9,7 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.4.2'
+    classpath 'com.android.tools.build:gradle:3.6.1'
```
Comment (aggarwalpulkit596) [inline build.gradle:12]: Don't update Gradle Tools

File: gradle/wrapper/gradle-wrapper.properties [hunk 5]
```diff
@@ -1,6 +1,6 @@
-#Tue Nov 19 15:14:08 IST 2019
+#Thu Mar 12 00:08:16 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```
Comment (aggarwalpulkit596) [inline gradle/wrapper/gradle-wrapper.properties:6]: Don't push changes made into the system files

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout-land/app_language_activity.xml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        app:layout_constraintTop_toBottomOf="@id/toolbar_shadow_view" />
+
+      <View
+        android:id="@+id/app_language_toolbar_shadow_view"
+        android:layout_width="match_parent"
+        android:layout_height="6dp"
+        android:background="@drawable/toolbar_drop_shadow" />
+    </FrameLayout>
+  </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout-land/app_language_activity.xml:62]: Add EOL

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (Kushal-Gera) [reply .idea/codeStyles/Project.xml:153]: how to make / revert these changes ?

Comment (rt4914) [reply .idea/codeStyles/Project.xml:153]: For simplicity you can copy of code of that file from here: https://github.com/oppia/oppia-android/tree/develop/.idea

And paste it back in your project and make a commit. Once that is done make sure you go to `Files Changed` section on your PR and see that the file no longer exists.

**Comment by rt4914** [comment]
@aggarwalpulkit596 Thanks for your review. But please do not request change or approve a PR instead just "Comment" on a PR.

**Comment by rt4914** [comment]
@Kushal-Gera Please finish the changes that have been suggested by @aggarwalpulkit596

**Comment by rt4914** [comment]
Closing this PR as we haven't heard from the author @Kushal-Gera .

**Pull request closed without merging**